src/cliargs/core.lua

```diff
@@ -149,7 +149,8 @@ local function create_core()
   ---        The number of columns assigned to the argument descriptions, set to
   ---        0 to auto set the total width to 72.
   function cli:set_colsz(key_cols, desc_cols)
-    colsz = { key_cols or colsz[1], desc_cols or colsz[2] }
+    colsz[1] = key_cols or colsz[1]
```

Comment:
I don't understand how these two statements aren't equivalent to the previous one? colsz is always a table of two items isn't it? What matters if we replace the reference with a new table instead of overwriting the existing table?